.travis.yml

```diff
@@ -14,18 +12,22 @@ cache:
     - $GOPATH/pkg/mod
 
 go:
-  - '1.11.x'
   - '1.x'
 
-install:
-  - go build -o sharkey-server -v ./server
-  - go build -o sharkey-client -v ./client
-  - docker build -t server:latest -f Dockerfile .
-  - docker build -t client:latest -f DockerfileClientTest .
+
+before_install:
+  - mysql -e 'CREATE DATABASE sharkey_test;'
+
+install: skip
```

Comment:
just curious why are we skipping this? I guess there's nothing to do if everything is in `script`?

Reply:
There's a default "install" step which installs your dependencies (via godep or other stuff).  We don't need that as we're using `go mod` to fetch on the fly.  It's probably harmless to just do nothing, but I wanted to make sure it wasn't messing with my build.